src/WebJobs.Script.WebHost/Security/Authorization/Policies/AuthorizationOptionsExtensions.cs

```diff
@@ -25,7 +25,7 @@ public static void AddScriptPolicies(this AuthorizationOptions options)
             options.AddPolicy(PolicyNames.SystemAuthLevel, p =>
             {
                 p.AddScriptAuthenticationSchemes();
-                p.AddRequirements(new AuthLevelRequirement(AuthorizationLevel.System));
+                p.AddRequirements(new ExtensionWebHookAuthorizationRequirement());
```

Comment:
There is already an existing NamedAuthLevelAuthorizationHandler that seems to be doing the same thing that the new one you're adding does. Perhaps the fix is to add `new NamedAuthLevelAuthorizationHandler(AuthorizationLevel.System)` here?

Reply:
Actually we can reuse NamedAuthLevelAuthorizationHandler